test/Suites.scala

```diff
@@ -0,0 +1,7 @@
+import integration.IntegrationTests
+import org.scalatest.Suites
+import test.ThriftUtilSpec
+
+class UnitTestsSuite extends Suites(new ThriftUtilSpec)
```

Comment:
Perhaps we could tag the integration tests instead? http://www.scalatest.org/user_guide/tagging_your_tests

That way we don't have to remember to add new unit tests to the suite